app/src/main/java/ch/epfl/sdp/drone3d/drone/DroneData.kt

```diff
@@ -19,6 +19,11 @@ interface DroneData {
      */
     data class CameraResolution(val width: Int, val height: Int)
 
+    /**
+     * A simple data class holding the values in millimeters of the camera sensor's size
+     */
+    data class SensorSize(val horizontalSize: Float, val VerticalSize: Float)
```

Comment:
VerticalSize could be verticalSize

Reply:
Agree with Squalene, should be lower case